### ISSUE TITLE
[HL2] [Feature Request] Add classic models, ragdolls, extras, DM

### ISSUE THREAD

**Issue by anonymous-guy1** [ValveSoftware/Source-1-Games#6612]
It would be great to have an option to use classic models, like the original vortigaunt and helicopter bomb models, before they were replaced by EP2 versions, similarly to how you added the classic effects option. Restoring the ragdoll physics from earlier versions would also be a nice touch.
Also the radioactive water effect seems broken as seen here and looks like that in my game also: https://www.reddit.com/r/HalfLife/comments/1gtc983/halflife_2_radioactive_water_2004_vs_2024/
It would be nice to include some beta content, concept art or similar in the extras tab in the main menu.
Also if HL2DM could be integrated into the main game, possibly with some new maps.
It's unfortunate that the native Mac version no longer works on newer devices due to the drop of 32-bit support. I understand 64bit might be too much work to make this possible.

On Steam, I noticed that Half-Life: Source doesn’t appear in search results since it’s delisted, while Half-Life Deathmatch: Source still shows up. This feels inconsistent.

I’m not sure if this is the right place for suggestions like these, but I wanted to share my thoughts somewhere :D. Thanks for all the amazing work on the 20th Anniversary update!

**Comment by nbmrjuhneibkr** [comment]
If Valve already went through the trouble of restoring original fire and blood effects, original models (vortigaunt especially) definitely need to be restored too.

Regarding visual downgrades and radioactive water effect - I'm also pretty sure that the airboat headlight used to be a lot brighter in original release (in its current state it's barely usable), but I can't verify this right now.

**Comment by dwartbg** [comment]
Yeah, since I don't want to repeat what you already said in another thread, I'll just say it here too:
- It would be amazing if they put the option to switch to the original Vortigaunt models, and overall other classic models and effects too.  Having Fire and Blood effects is OK, but there are numerous other classic effects that can be incorporated so we have the ultimate HL2 version with absolutely everything that has ever been inside the game since it's original release.
